packages/microsoft_kiota_http/example/pubspec.yaml

```diff
@@ -0,0 +1,11 @@
+name: microsoft_kiota_http_example
```

Comment:
should we move the example outside of the http package structure?

Reply:
No that is part of the Dart package layout. Each package _should_ come with an example on how to use it. In theory, we should also add examples to all other packages.

Reply:
Thanks for explaining this to me. :)